### ISSUE TITLE
Extra window created

### ISSUE THREAD

**Issue by MikaelStalvik** [opentk/GLWpfControl#111]
When setting up the most simple sample WPF application with .NET 7 a ghost window with the title "OpenTK Window" will be created parallell to the MainWindow.

This extra window is visible on the taskbar as well as from the Task Manager when expaning the process.

Changing to the extra window doesn't do much but can sometimes leave traces of contents on it.

**Expected result**
No extra window shall be created.

**Configuration used**
- OpenTK 4.8.0
- OpenTK.GLWpfControl 4.2.3
- .NET7
- Windows 11

**Sample code**
_MainWindow.xaml_
`<Window x:Class="OpenTkWpf7.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:local="clr-namespace:OpenTkWpf7" xmlns:glWpfControl="clr-namespace:OpenTK.Wpf;assembly=GLWpfControl"
        mc:Ignorable="d"
        Title="MainWindow" Height="450" Width="800">
    <Grid>
        <glWpfControl:GLWpfControl 
            x:Name="OpenTkControl"             
            Render="OpenTkControl_OnRender"/>    
    </Grid>
</Window>
`

_MainWindow.xaml.cs_
```
public partial class MainWindow : Window
    {
        public MainWindow()
        {
            InitializeComponent();
            var settings = new GLWpfControlSettings
            {
                MajorVersion = 3,
                MinorVersion = 1,
            };
            OpenTkControl.Start(settings);
        }

        private void OpenTkControl_OnRender(TimeSpan obj)
        {
        }
    }

```

**Comment by Svabik** [comment]
I have to add to this - I was observing that window in the past too (previous versions of GLWpfControl and OpenTK). But it was visible only when my application started a function using full CPU - making UI irresponsive. The black rectangle appeared in top left screen corner.
Last week I have upgraded the OpenTK Only - and since then I observe exactly what MikaelStalvik describes. Very annoying.

**Comment by YamingZ** [comment]
I also encountered this extra window called OpenTK Window, is there any way to remove it?

**Comment by MikaelStalvik** [comment]
After some digging, it seems like this happens becuase of some change in OpenTK itself, not this control.
Reverting to OpenTK 4.7.7removes the problem.

Possibly this issue needs to be raised for the OpenTK control itself?

**Comment by NogginBops** [comment]
I think that #115 should solve this issue, as the window visibility bug should be fixed in `4.8.0+`.

**Comment by MikaelStalvik** [comment]
Yes, version 4.8.1 seems to have resolved this. Thanks!
